.github/workflows/benchmarks.yml

```diff
@@ -49,16 +49,30 @@ jobs:
           ./integration-tests/scripts/baseline/main.py --test integration-tests/perf-all.json \
             | sort \
             | awk -f ./integration-tests/scripts/baseline/format.awk > benchmark.md
-
-          delimiter=$(openssl rand -hex 8)
-          {
-            echo "PERF_TABLE<<${delimiter}"
-            cat benchmark.md
-            echo "${delimiter}"
-          } >> "$GITHUB_ENV"
         env:
           GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT: "${{ secrets.GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT }}"
 
+      - name: Plot benchmark graphs
+        run: |
+          python3 -m pip install -r utilities/perf-plotter/requirements.txt
```

Comment:
Consider moving this to the `Install python deps` step, to keep all the dependency setup in the same place